lighthouse-core/report/html/report-styles.css

```diff
@@ -438,6 +438,16 @@
 .lh-audit--error .lh-audit__score-icon {
   display: none;
 }
+/* This styling should be listed after pass/average/fail styling for the same elementss, so that it takes precedence */
```

Comment:
this makes me think we shouldn't apply the `pass/average/fail` styling when it's informative :)

Reply:
Sounds good to me :) Updated.